toolbox/util/githubUtils.go

```diff
@@ -180,3 +188,23 @@ func CreateCommitPushToRemote(branch, commitMsg string) error {
 	_, err := Shell("git push -f --set-upstream origin " + branch)
 	return err
 }
+
+// BlockMergingOnBranch adds "do-not-merge/post-submit" labels to all PRs
+// in :branch on the repo which the :githubClnt connects to
+func BlockMergingOnBranch(githubClnt *GithubClient, repo, branch string) error {
```

Comment:
Maybe change this to write /hold on the PR ? that way people know what s going on and it is not implementation specific

Reply:
it hold/post-submit-failure supported? that way it won't conflict with other uses of /hold

Reply:
Unfortunately hold/post-submit-failure is not supported. Using /hold cancel directly might unblock PRs that are intended to be held off. I will sync up with Yutong to understand if we could define our own label on mungegithub and how that is going to affect the migration to tide. For now lets keep it as it.